.github/workflows/SQL2019.yaml

```diff
@@ -0,0 +1,175 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+#
+# See https://github.com/r-lib/actions/tree/master/examples#readme for
+# additional example workflows available for the R community.
+
+name: BuildAndTest
```

Comment:
Can you make all the names that display for GitHub actions for this pipeline more informative? here they look very vague:
- **BuildAndTest** -> build and test what? R 3.5.2 and Python 3.7.<something>
- **SQL19**  -> SQL Server 2019

![image](https://github.com/microsoft/sqlmlutils/assets/6414189/56f6ee6a-9a94-467d-bd9e-8e1d5ba2ad5b)
![image](https://github.com/microsoft/sqlmlutils/assets/6414189/56f6ee6a-9a94-467d-bd9e-8e1d5ba2ad5b)

Reply:
Updated to `BuildAndTest SQLMLUtils for SQL Server 2019`